apps/language_server/lib/language_server/providers/execute_command.ex

```diff
@@ -3,29 +3,31 @@ defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand do
   Adds a @spec annotation to the document when the user clicks on a code lens.
   """
 
-  @callback execute([any], %ElixirLS.LanguageServer.Server{}) ::
-              {:ok, any} | {:error, atom, String.t()}
-
-  def execute(command, args, state) do
-    handler =
-      case command do
-        "spec:" <> _ ->
-          ElixirLS.LanguageServer.Providers.ExecuteCommand.ApplySpec
+  alias ElixirLS.LanguageServer.Providers.ExecuteCommand, as: C
```

Comment:
Can we just do a normal alias here. I think that helps with readability/clarity.
```suggestion
  alias ElixirLS.LanguageServer.Providers.ExecuteCommand
```

Reply:
done